### ISSUE TITLE
This Plugin so much better than the "floatinghotpot/cordova-admob-pro" but we need more mediation support.

### ISSUE THREAD

**Issue by geekyguy18** [ratson/cordova-plugin-admob-free#62]
I was using this AdMob Pro Plugin, but it seems that even you buy licence the start steeling impressions once app is popular enough, having a very bad experience with it.

Anyways Thanks to this great guy @becvert who created two mediation plugins for this plugin

[cordova-plugin-admob-mediation-facebook](https://github.com/becvert/cordova-plugin-admob-mediation-facebook)
[cordova-plugin-admob-mediation-adbuddiz](https://github.com/becvert/cordova-plugin-admob-mediation-adbuddiz)

I would like to request every one who ever used pro plugins and pissed off like me, that we should build more mediation plugin together so that people stop loosing their money with pro plugins. I wanted to create mediation plugin myself but as i am not native android developer i don't know java & android stuff. if making mediation plugin without them possible then kindly guide me how can i contribute to this development. I think Mobfox, Inmobi & Mopub mediation should developed next. If anyone interested in that kindly comment here. @becvert we need your guide here.

**Comment by geekyguy18** [comment]
Update : Great seems like it was too easy to build mediation plugin, i got it working, will be update soon here :)

**Comment by geekyguy18** [comment]
Update 1: Mobfox Banners and Interstitial ads are working now :)

**Comment by ratson** [comment]
Thanks for creating the plugin, I have linked you project in https://github.com/rehy/cordova-admob-mediation/tree/master/packages/cordova-admob-mobfox

I will keep this open for discussion.

**Comment by geekyguy18** [comment]
Thank you @ratson for linking, btw i did not understand how [this thing](https://github.com/rehy/cordova-admob-mediation/tree/master/packages/cordova-admob-mobfox) will work if there is no adapter and sdk included in your repo, did you forget to put there?

**Comment by ratson** [comment]
@geekyguy18 No, it is by design not committing the binary files, see https://github.com/rehy/cordova-admob-mediation#contributing

**Comment by geekyguy18** [comment]
Update 2: I have created another AdMob Mediation plugin for **Adcash** here:
https://github.com/geekyguy18/cordova-plugin-admob-mediation-adcash

Kindly add this one too in your list, i hope it will help someone.

**Comment by rodrigograca31** [comment]
:+1: 
great to see that we can do facebook mediation!
Will be adding FAN to my project later this week.
